### ISSUE TITLE
[Probable Bug] Pet Quality numbers have shifted

### ISSUE THREAD

**Issue by MechaZao** [axc450/pbs#99]
Sorry, this may not be an actual issue but I'm not really sure of a good resource to ask for script help.

my scripts, which I have used for years, are not working now. They worked properly last time I played in Dragonflight.
I'm not sure if something is wrong, or if I'm doing something wrong.
```
--Quit if no Rares
quit [enemy(#1).quality!=4 & enemy(#2).quality!=4 & enemy(#3).quality!=4]
quit [enemy(#1).quality!=4 & enemy(#2).quality!=4 & !enemy(#3).exists]
quit [enemy(#1).quality!=4 & !enemy(#2).exists & !enemy(#3).exists]
test(Rare pet found!)
```
this should only quit if there are no Rare enemy pets found, however instead it quits every single time.
and in another script, my pets killed a blue quality pet despite
```
catch [enemy.quality>3]
```

its almost as if the quality is being ignored or returned incorrectly?
again, if I've made a mistake here please let me know!

including a screenshot of the script editor test result being "quit" despite there being a rare enemy in slot 3.
![image](https://github.com/user-attachments/assets/f05c4c2f-afe2-4e1f-b5d3-145983a1f753)

**Comment by MechaZao** [comment]
I have done further testing and discovered that the values for quality appear to have changed at some point recently.

I'm now seeing these results: 

Poor: <1 (0 returned an error)
Common: 1
Uncommon: 2
Rare: 3
Epic: 4
Legendary: 5

I came to this conclusion by running this script on various enemy types:
```
if [enemy.quality=x]
test(success!)
ei
``` 

I believe this is now a bug of some sort, so I've edited my issue title.

**Comment by bloerwald** [comment]
Should be fixed by v1.10.6.
